test/apps/helper.ts

```diff
@@ -0,0 +1,16 @@
+// FIXME: move this to a test helper that can be used by other apps
+
+import cacheManager from 'cache-manager'
+import { Application } from '../../src'
+
+const cache = cacheManager.caching({ store: 'memory', ttl: 0 })
```

Comment:
The `cache-manager` types required a `ttl` property. I set this to 0 seconds - if this should be a different value, let me know.